gnomad_constraint/utils/constraint.py

```diff
@@ -734,6 +734,52 @@ def calculate_mu_by_downsampling(
     return annotate_mutation_type(ht)
 
 
+def add_oe_lof_upper_rank_and_bin(
+    ht: hl.Table, use_mane_select_instead_of_canonical: bool = False
+) -> hl.Table:
+    """
+    Compute the rank, decile, and sextile of the lof oe upper confidence interval for MANE Select or canonical ensembl transcripts.
+
+    :param ht: Input Table with the value for the lof oe upper confidence interval stored in ht.lof.oe_ci.upper.
+    :param use_mane_select_instead_of_canonical: Use MANE Select rather than canonical transcripts for filtering the Table.
```

Comment:
```suggestion
    :param use_mane_select_instead_of_canonical: Use MANE Select rather than canonical transcripts for filtering the Table. Default is False.
```
though wouldn't we want the default to be True? is this set to False to maintain backwards compatibility?